### PR TITLE
feat: update xSushi price feed aip

### PR DESCRIPTION
**Pull request by rustboyar**

### CODE REVIEW

File: content/aips/XSUSHI_PRICE_FEED_SWAP.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+title: Update xSushi price feed on Aave V2
+author: BGD Labs (@bgdlabs)
+shortDescription: Swap xSushi price feed on Aave V2 pool on Mainnet
+discussions: TBA
+created: 2023-02-10
+---
+
+## Simple Summary
+
+This proposal changes the current price feed for xSushi to ChainLink's oracle for Aave V2 on Mainnet.
```
Comment (eboadom) [inline content/aips/XSUSHI_PRICE_FEED_SWAP.md:11]:
```suggestion
This proposal changes the current price feed for xSushi (adapter) to a Chainlink calculated oracle, on Aave V2 Ethereum.
```

File: content/aips/XSUSHI_PRICE_FEED_SWAP.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## Motivation
+
+Currently, the custom [price feed](https://etherscan.io/address/0x9b26214bEC078E68a394AaEbfbffF406Ce14893F) is used for xSushi, which calculates the price based on `SUSHI / ETH` oracle and `SUSHI / xSUSHI` ratio on-chain. It would be way more effective in terms of technical support and gas cost to use the [feed](https://etherscan.io/address/0xF05D9B6C08757EAcb1fbec18e36A1B7566a13DEB) deployed by Chainlink, which does this calculations off-chain.
```
Comment (eboadom) [inline content/aips/XSUSHI_PRICE_FEED_SWAP.md:15]:
```suggestion
Currently, a custom [price feed](https://etherscan.io/address/0x9b26214bEC078E68a394AaEbfbffF406Ce14893F) is used for xSushi, which calculates the price based on `SUSHI / ETH` oracle and `SUSHI / xSUSHI` ratio on-chain. It is more effective in terms of technical support and gas cost to use the [feed](https://etherscan.io/address/0xF05D9B6C08757EAcb1fbec18e36A1B7566a13DEB) deployed by Chainlink, which does this calculations off-chain. In addition, any risk of on-chain manipulation, even if not applicable at the moment with the asset not enabled for borrowing, gets removed.
```

File: content/aips/XSUSHI_PRICE_FEED_SWAP.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+## Specification
+
+Upon execution the proposal will:
+
+- call `ORACLE.setAssetSources(0x8798249c2E607446EfB7Ad49eC89dD1865Ff4272, 0xF05D9B6C08757EAcb1fbec18e36A1B7566a13DEB)` to replace the price source for xSushi
+
+## Security and additional considerations
+
+We applied the following security procedures for this upgrade:
```
Comment (eboadom) [inline content/aips/XSUSHI_PRICE_FEED_SWAP.md:25]:
```suggestion
To have extra assurances on the feed swap, we did the following:
```

File: content/aips/XSUSHI_PRICE_FEED_SWAP.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+- **Historical Feeds Comparison**: Comparing the answers from the current feed with the new one for the last year with a 2-hour step gave the following results:
+  - 24.88% of results differ for less than 0.1%
+  - 55.63% have between 0.1% and 1% difference
+  - 19.49% is between 1% and 2%
+
```
Comment (eboadom) [inline content/aips/XSUSHI_PRICE_FEED_SWAP.md:31]: You can add that that deviations are quite solid consider the specific deviation % of the feed

**Review by sakulstra — APPROVED** [review]

**Pull request merged**